### DISCUSSION TITLE
ISO Contains a UEFI Bootloader That has Been Revoked

### DISCUSSION THREAD

**Discussion by WhoIsConch** [Nitrux/nitrux-bug-tracker#137] [Q&A] [answered]
Hi there! Today, I decided to try out using Nitrux. It worked okay in Virtualbox, but I wanted to try out Nitrux Live on my laptop. However, when flashing the ISO to Rufus I got an error that reads:

> Rufus detected that the ISO you have selected contains a UEFI
bootloader that has been revoked and that will produce a
•Security Violation• screen, when Secure Boot is enabled on a
fully up to date LIEFI system,

And, sure enough, when trying to boot from the USB on my laptop, I get a security violation error. I just wanted to know what was up with this and what my next step would be if I still wanted to use Nitrux on a PC. Thanks!

**Comment by UriHerrera** [comment, marked as answer]
As noted in the tutorial about [installing the distribution](https://nxos.org/tutorial/how-to-install-nitrux/#flash-iso-with-rufus).

> ⚠️ Important: [Rufus 4.2.4070 Beta](https://www.neowin.net/software/rufus-424070-beta/) introduced a feature to the software to display a warning about revoked UEFI bootloaders, "Add detection and warning for UEFI revoked bootloaders (including ones revoked through SkuSiPolicy.p7b)." This will cause Rufus to display a warning message. Users can ignore this warning as the bootloader post-installation is signed; however, the bootloader in the ISO isn't signed, nor the Liquorix kernel is signed, and the distribution will not boot with Secure Boot enabled; see [System Requirements](https://nxos.org/tutorial/how-to-install-nitrux/#system-requirements).

> See [this issue at the Rufus repository](https://github.com/pbatard/rufus/issues/2244) for context about this warning.
> Also, [see this reply in a Reddit thread of the Rufus developer](https://www.reddit.com/r/ghostspectre/comments/15un0rq/trying_to_download_ghost_for_first_time_so/) about why that was implemented.

---

TL;DR

It's a new "feature" in Rufus that caters to Microsoft's Secure Boot infrastructure.

**Reply by MNLierman** [reply]
It's ASTONISHING that we're over a year out and the official DISM and ISO creator tools for creating an unattended Windows 11 ISO, pulled straight from Aria2 contains 5 revoked certs. I don't think this will boot on my laptop, as I'm pretty positive that it's taken the update during the last "unsafe" install. I watched Win 11 24H2, an insecure one, patch the BIOS during install, not even joking, I have a video saved of it doing it. Patched itself because it knew it was an insecure revoked bootloader? To prevent me from ever doing it again? Thanks Microsoft for patching my BIOS but not releasing updating ISO and DISM tools. Assholes. The place ran better with Gates around.
